{
  "comments": [
    {
      "key": {
        "uuid": "c2f85f84_f1f31811",
        "filename": "certService/src/main/java/org/onap/aaf/certservice/api/CertificationService.java",
        "patchSetId": 2
      },
      "lineNbr": 72,
      "author": {
        "id": 5775
      },
      "writtenOn": "2020-02-10T14:34:18Z",
      "side": 1,
      "message": "Exception occurred... \n\nWhy TRACE level, not ERROR?",
      "revId": "f85918c4c0c6cada28d7384bad63396225751868",
      "serverId": "14b4e41f-c4e3-4fb9-9955-6a9b5656895a",
      "unresolved": true
    }
  ]
}